app/App/Panel/Main/Account/Requests/TransactionRequest/TxApproval/approvals/OtherChain/index.js

```diff
@@ -13,26 +13,24 @@ class TxApproval extends React.Component {
     }
   }
 
-  decline (req) {
-    link.rpc('declineRequest', req, () => {})
+  approve () {
+    if (this.props.allowOtherChain) this.props.allowOtherChain()
```

Comment:
we shouldn't allow this component to even be constructed without this as it has no function in that case and can never be accepted